README.md

```diff
@@ -107,9 +107,9 @@ source .venv/bin/activate
 ```sh
 pip install -r requirements.txt
 ```
-4. Finally, to build the Jupyter Book
+4. Finally, to build the sphinx-build
```

Comment:
we can still call it "Jupyter Book" or if we prefer "site". "sphinx-build" doesn't really mean anything other than the terminal command.

Re jupyter book, we are only changing a step in the build process which has no effect on the result nor most of the build process (we still use the jupyter_book extension for example, the `_toc.yml` instead of `toctree` directives embedded within the content...)